ServerCore/Pages/Teams/Details.cshtml

```diff
@@ -10,6 +10,30 @@
 }
 @using Microsoft.AspNetCore.Http.Extensions @* for GetEncodedUrl *@
 
+@{
+    var mailtoUrl = "mailto:";
```

Comment:
Possible to make this a StringBuilder and Append() all these things? It's somewhat more efficient to do it that way.